src/Illuminate/Log/composer.json

```
@@ -17,7 +17,7 @@
         "php": "^7.2",
         "illuminate/contracts": "^6.0",
         "illuminate/support": "^6.0",
-        "monolog/monolog": "^1.11"
+        "monolog/monolog": "^1.12 || ^2.0"
```

Please use `^1.12|^2.0` notation for constraints.

```suggestion
        "monolog/monolog": "^1.12|^2.0"
```

---

done